IT/src/test/java/com/microsoft/gctoolkit/integration/collections/XYDataSet.java

```diff
@@ -0,0 +1,100 @@
+package com.microsoft.gctoolkit.integration.collections;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.OptionalDouble;
+import java.util.stream.Stream;
+
+public class XYDataSet {
```

Comment:
Another unfortunate duplication of code.

Reply:
IMO, this should have been left in the API but it was pulled out. I'm open for suggestions because this isn't the only place where it's going to be duplicated